scripts/validate-helm.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+if [ -L ${BASH_SOURCE-$0} ]; then
+  PWD=$(dirname $(readlink "${BASH_SOURCE-$0}"))
+else
+  PWD=$(dirname ${BASH_SOURCE-$0})
+fi
+export CURRENT_PATH=$(cd "${PWD}" >/dev/null; pwd)
+export KUBERAY_HOME=${CURRENT_PATH}/..
+
+# Install kubeconform
+KUBECONFORM_INSTALL=$(mktemp -d)
+trap 'rm -rf "${KUBECONFORM_INSTALL}"' EXIT
+curl -L https://github.com/yannh/kubeconform/releases/latest/download/kubeconform-linux-amd64.tar.gz | tar xz -C "${KUBECONFORM_INSTALL}"
+curl -o "${KUBECONFORM_INSTALL}/crd2schema.py" https://raw.githubusercontent.com/yannh/kubeconform/master/scripts/openapi2jsonschema.py
+SCRIPT_PATH="${KUBECONFORM_INSTALL}/crd2schema.py"
```

Comment:
We can consider store this script in our repo so that we don't need to download it everytime.